### ISSUE TITLE
Fix epydoc error in trial._dist

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5995]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)| @therve reported|
|-|-|
|Trac ID|trac#5995|
|Type|defect|
|Created|2012-09-17 12:25:08Z|
|Branch|https://github.com/twisted/twisted/tree/trial-epytext-5995|

I introduced a epytext error when merging [#1784](https://github.com/twisted/twisted/issues/1784):

```
found unknown field on 'twisted.trial._dist.disttrial.DistTrialRunner.launchWorkerProcesses': <Field 'arguments' None 'Extra arguments pas....'>
```

It needs to be fixed, maybe along some other ones for karma.

<details><summary>Searchable metadata</summary>

```
trac-id__5995 5995
type__defect defect
reporter__therve therve
priority__low low
milestone__ 
branch__branches_trial_epytext_5995 branches/trial-epytext-5995
branch_author__therve therve
status__closed closed
resolution__fixed fixed
component__trial trial
keywords__easy easy
time__1347884708000000 1347884708000000
changetime__1347889565000000 1347889565000000
version__None None
owner__therve therve
cc__jml
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_2"></a>|@therve commented|
|-|-|

(In [35719]) Branching to 'trial-epytext-5995'

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_3"></a>|@therve removed owner|
|-|-|

I fixed the mentioned and some other ones in trial.

Build results here: http://buildbot.twistedmatrix.com/builders/documentation/builds/2515

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set owner to @therve|
|-|-|

Thanks.  Please add a news fragment and merge.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_5"></a>|@therve set status to `closed`|
|-|-|

(In [35722]) Merge trial-epytext-5995

Author: therve
Reviewer: exarkun
Fixes: [#5995](https://github.com/twisted/twisted/issues/5995)

Remove some epytext errors in twisted.trial
